rasa/core/policies/ted_policy.py

```diff
@@ -710,7 +711,11 @@ def train(
             )
             return self._resource
 
-        self.run_training(model_data, label_ids)
+        if self.config[USE_GPU]:
+            self.run_training(model_data, label_ids)
+        else:
+            with tf.device('/cpu:0'):
+                self.run_training(model_data, label_ids)
```

Comment:
As explained in the previous comments, please use a nullcontext to avoid code duplication:
```suggestion
        with( tf.device('/cpu:0') if self.config[USE_GPU] else self.config[USE_GPU]nullcontext()) ):
            self.run_training(model_data, label_ids)
```